flink/v1.17/flink/src/test/java/org/apache/iceberg/flink/SimpleDataUtil.java

```diff
@@ -349,7 +349,10 @@ public static void assertTableRecords(String tablePath, List<Record> expected, S
 
   public static StructLikeSet expectedRowSet(Table table, Record... records) {
     StructLikeSet set = StructLikeSet.create(table.schema().asStruct());
-    Collections.addAll(set, records);
+    InternalRecordWrapper wrapper = new InternalRecordWrapper(table.schema().asStruct());
```

Comment:
Why do we need this wrapper?

Reply:
`StructLikeSet ` use a `StructLikeWrapper` to wrap every input record, and it converts the TimeStamp record to long (Iceberg uses long to represent TimeStamp). So here we use `InternalRecordWrapper ` to convert the external StruckLike record to an internal record first.

Reply:
I ran the UT without the `InternalRecordWrapper` and the result is as follows:
```
java.lang.AssertionError: Should have expected records 
Expected :[Record(1, 2023-06-18T13:14:24.620+08:00)]
Actual   :[Record(1, 2023-06-18T05:14:24.620Z, 0)]
```

It seems the time values (long values) are the same but `actualRowSet` doesn't apply the correct zone offset.  Applying  `InternalRecordWrapper` do fix this.